integration/kind-test/scripts/run-tests.sh

```diff
@@ -40,13 +44,24 @@ if [ "$exit_code" -eq 0 ] ; then
     exit $?
   fi
 
-  go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT "$updated_endpoints"
+  go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT $SERVICE_TYPE "$updated_endpoints"
   exit_code=$?
 
   if [ "$exit_code" -eq 0 ] ; then
     ./integration/shared/scripts/test-import.sh "$UPDATED_ENDPOINT_COUNT" "$updated_endpoints"
     exit_code=$?
   fi
+
+  if [ "$exit_code" -eq 0 ] ; then
+    ./integration/kind-test/scripts/DNS-test.sh
+    exit_code=$?
+  fi
+fi
+
+# Scale deployment back down for future test and delete service export
+if [ "$exit_code" -eq 0 ] ; then
+  $KUBECTL_BIN scale deployment/"$deployment" --replicas="$EXPECTED_ENDPOINT_COUNT" --namespace "$NAMESPACE"
```

Comment:
Not too sure if it's worth scaling down here, or if so to only do it if tests succeed.